.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+name: Mark and close stale issues
+
+on:
+  schedule:
+    - cron: '0 3 * * *' # daily at 03:00 UTC
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: read
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run stale action (issues only)
+        uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          operations-per-run: 200
+
+          # Issues policy
+          days-before-issue-stale: 14
+          days-before-issue-close: 14
+          stale-issue-label: 'stale'
```

Comment:
I believe you need to manually add this tag first otherwise it'll error (at least other actions dealing with label do)